seeker/message-center.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
```

Comment:
Is it possible to rename this file to `messages.html` instead? Because currently the header tries to point to `messages.html`, so we have to update one or the other.